Aptoide/aptoide-client-v8
# [feature/MOB-127] Install On TV Dark Theme

**What does this PR do?**

   Adds support for dark theme in install on tv dialog (app view).

**Database changed?**

   No

**How should this be manually tested?**

  Check the install on tv dialog on app view for dark and light theme and check that it's ok.

**What are the relevant tickets?**

  [ASV-1763](https://aptoide.atlassian.net/browse/ASV-1763)

**Code Review Checklist**
